package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -476,6 +477,7 @@ dhcp_add() {
 	config_get leasetime "$cfg" leasetime 12h
 	config_get options "$cfg" options
 	config_get_bool dynamicdhcp "$cfg" dynamicdhcp 1
+	config_get tag_list "$cfg" tags
```

Comment:
One minor change; move definition of local tags to the beginning of the function dhcp_add